src/store/main-store/MainMutations.ts

```diff
@@ -6,11 +6,19 @@ export default class MainMutations extends Mutations<MainState> {
         this.state.testData = !this.state.testData;
     }
 
-    public changeLanguajeToSpanish(): void {
+    public changeLanguageToSpanish(): void {
         this.state.currentLanguage = 'es';
     }
 
-    public changeLanguajeToEnglish(): void {
+    public changeLanguageToEnglish(): void {
         this.state.currentLanguage = 'en';
     }
+
+    public activateMaintenancePage(): void {
```

Comment:
@gonzalo-plaza  I am not shure if activate o active... maybe @PVillasana  and @JuanJosePG  cand say how is better

Reply:
@PVillasana @JuanJosePG  @pedrojgm

Reply:
The first time I wrote the code I had "active" but then I changed to "activate",  I'm not sure if it's the correct way but looks fine for me.